README.md

```diff
@@ -1,5 +1,15 @@
 # HRC - Styleguide
 
+## Install
+
+1. Setup your virtualhost (like `http://hrc.dev`) to serve `/public`.
```

Comment:
Normally, BrowserSync (`gulp serve`) will do that, no need of Apache or even NodeJS dedicated server 😉